java/splitter/splitter-core/src/main/java/sleeper/splitter/FindPartitionsToSplit.java

```diff
@@ -60,13 +60,15 @@ public FindPartitionsToSplit(
     }
 
     public void run() {
-        tablePropertiesProvider.streamOnlineTables().forEach(tableProperties -> {
-            try {
-                findPartitionsToSplit(tableProperties, stateStoreProvider.getStateStore(tableProperties));
-            } catch (StateStoreException e) {
-                LOGGER.error("StateStoreException thrown whilst running FindPartitionsToSplit", e);
-            }
-        });
+        tablePropertiesProvider.streamOnlineTables().forEach(tableProperties -> run(tableProperties, stateStoreProvider.getStateStore(tableProperties)));
```

Comment:
It looks like we need to remove this method, and maybe take an InvokeForTableRequest instead?